### ISSUE TITLE
County-specific logic: Oakland County forms

### ISSUE THREAD

**Issue by normon66** [mplp/docassemble-MLHPPOAndProposedOrder#129]
1. Always generate Contact Information Sheet (this is current functionality)
2. Once using ETC, the Petition should be included behind a single-page cover sheet, seen below. Note that the NOT AN EXPARTE PETITION message should only be shown when that is the case. In other words, it does NOT need to say EX PARTE PETITION.
   ![image](https://github.com/user-attachments/assets/76445dc8-d191-4d3f-bc63-ded4fc4cd948)

**Comment by ekressmiller** [comment]
@RamiLorca The ETC emails are currently set up in the dev server config to come to my email inbox. So feel free to collaborate with me if you want to test when you implement this. Or we can change it temporarily to your address if that would be too annoying.

**Comment by ekressmiller** [comment]
@RamiLorca: actually please WAIT on this until I am able to check in with Brett.

**Comment by ekressmiller** [comment]
@RamiLorca this issue now **only** covers always generating the contact information sheet for Oakland. If that's already done, you can feel free to close this.  I'll make a separate issue for the coversheet.

**Comment by RamiLorca** [comment]
@ekressmiller I should have asked this before closing out the issue, but I just wanted to confirm if we still want the Oakland county al_court_bundle to include a cover sheet. I believe that used to also be part of this issue.

**Comment by ekressmiller** [comment]
@RamiLorca I created a separate issue for that (Oakland doesn't need a coversheet but the other counties do), and I think it's basically working: https://github.com/mplp/docassemble-MLHPPOAndProposedOrder/issues/160
